src/overlays/actors/ovl_En_Yabusame_Mark/z_en_yabusame_mark.h

```diff
@@ -6,9 +6,19 @@
 
 struct EnYabusameMark;
 
+typedef void (*EnYabusameMarkActionFunc)(struct EnYabusameMark*, GlobalContext*);
+
 typedef struct EnYabusameMark {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0xC4];
+    /* 0x014C */ EnYabusameMarkActionFunc actionFunc;
+	/* 0x0150 */ s16 typeIndex;
+	/* 0x0152 */ s16 subTypeIndex;
+	/* 0x0154 */ Vec3f worldPos;
+	/* 0x0160 */ Vec3f vertexA;
+	/* 0x016C */ Vec3f vertexB;
+	/* 0x0178 */ Vec3f vertexC;
+	/* 0x0184 */ Vec3f vertexD;
+	/* 0x0190 */ ColliderQuad collider;
```

Comment:
I think some tab characters were inserted here, which should be replaced with proper spaces, ideally